components/Calendar/CalendarWorkoutCard.jsx

```diff
@@ -66,7 +66,7 @@ const CalendarWorkoutCard = (props) => {
 
   SetRow.propTypes = {
     index: PropTypes.number.isRequired,
-    lbs: PropTypes.number,
+    lbs: PropTypes.any,
```

Comment:
Just curious why did we need to make this change?

Reply:
To allow 'N/A' as a value